### DISCUSSION TITLE
azure 安裝pyzbar的問題

### DISCUSSION THREAD

**Discussion by pocper1** [1101-GS3073-C/Python-Community-GS3073-C#174] [Q&A]
![azure 套件問題](https://user-images.githubusercontent.com/76590438/148186722-3f5a5877-b93f-4497-adf8-a11970ef80a1.png)

在使用pyzbar當中的zbar 一直出現安裝問題，不知道要如何解決呢？

**Comment by chia-kai-chang** [comment]
Azure Web App是Linux作業系統，Zbar是要另外安裝的：https://howtoinstall.co/en/zbar-tools

你可以試試從Azure Web App左側的SSH連到Linux的命令列，嘗試安裝。但如果還是不行的話，可以參考以下變通方式：

另外租用雲端虛擬主機(Azure, AWS, GCP, Vultr，個人偏好Vultr.com)，依序安裝必要的軟體( https://forum.labview360.org/t/topic/31400 ) 並設定反向代理 ( https://forum.labview360.org/t/topic/31428 )

這樣將擁有最高的作業系統安裝軟體權限，但是缺點是每個月要花錢。


你先試試看使用SSH連到Azure Web App的Linux主機能否安裝zbar，如果不行再回文討論。也許可以多個組放在同一台主機中。
